standards/deployment.md

```diff
@@ -0,0 +1,14 @@
+# <a name="deployment"></a>Deployment
+
+There are really 2 primary ways to handle deployment on Rails projects at The Nerdery:
```

Comment:
A comment on writing style, feel free to ignore if you disagree.

Would this be better worded as 'There are two preferred ways' instead of 'There are really 2 primary ways'?